docs/specs/clients/sign/client-api.md

```diff
@@ -74,6 +74,50 @@ abstract class Client {
     reason: Reason;
   }): Promise<void>;
 
+  // ---------- Wallet Authentication Methods ----------------------------------------------- //
+
+  // for proposer to request wallet authentication
+  public abstract sessionAuthenticate(params: {
+    chains: string[];
+    domain: string;
+    nonce: string;
+    aud: string;
+    type?: CacaoHeader["t"];
+    nbf?: string;
+    exp?: string;
+    statement?: string;
+    requestId?: string;
+    resources?: string[];
+    expiry?: number;
+    pairingTopic?: string;
+    methods?: string[];
```

Comment:
`methods` being optional allow authenticated session without ability to send requests. is it sth that we aim for?